webpack.config.ts

```diff
@@ -62,6 +62,9 @@ const config = (env: EnvConfig): Configuration => {
         },
         resolve: {
             extensions: ['.tsx', '.ts', '.js'],
+            alias: {
+                'Styles': path.resolve(__dirname, 'src/styles'),
```

Comment:
I thought we are going to maintain the alias in one place - tsconfig.json after we added `TsconfigPathsPlugin`

Reply:
Ah yes, this was before I pulled the latest code from @farrah-deriv 👍 Let me remove that.

Reply:
Will copy cover the config from her PR 🙏